lib/spack/spack/reporters/cdash.py

```diff
@@ -42,8 +42,14 @@
 }
 
 # Initialize data structures common to each phase's report.
-cdash_phases = set(map_phases_to_cdash.values())
-cdash_phases.add("update")
+CDASH_PHASES = set(MAP_PHASES_TO_CDASH.values())
+CDASH_PHASES.add("update")
+
+
+CDashConfiguration = collections.namedtuple(
+    "CDashConfiguration",
+    ["upload_url", "packages", "build", "site", "buildstamp", "track", "ctest_parsing"],
```

Comment:
Here I maintained the same semantic for `ctest_parsing` as before[^1], but I see no place where we have such an argument defined. If this is a leftover, and I didn't overlook some definition, I can remove a few lines more.

[^1]: In fact the entire PR is just restructuring code, without changing observed behavior (or at least that is the aim).

Reply:
Maybe when @zackgalbreath gets back he can clarify this but I think it's good for now.